transaction.go

```diff
@@ -6,22 +6,24 @@ import "time"
 // For more details see https://developers.xendit.co/api-reference/#transactions.
 // For documentation of subpackage payout, checkout https://pkg.go.dev/github.com/xendit/xendit-go/transaction
 type Transaction struct {
-	ID                string         `json:"id"`
-	ProductID         string         `json:"product_id"`
-	Type              string         `json:"type"`
-	ChannelCode       string         `json:"channel_code,omitempty"`
-	ReferenceID       string         `json:"reference_id"`
-	AccountIdentifier string         `json:"account_identifier,omitempty"`
-	Currency          string         `json:"currency,omitempty"`
-	Amount            float64        `json:"amount"`
-	NetAmount         float64        `json:"net_amount"`
-	Cashflow          string         `json:"cashflow"`
-	Status            string         `json:"status"`
-	ChannelCategory   string         `json:"channel_category"`
-	BusinessID        string         `json:"business_id"`
-	Created           *time.Time     `json:"created"`
-	Updated           *time.Time     `json:"updated"`
-	Fee               TransactionFee `json:"fee"`
+	ID                      string         `json:"id"`
+	ProductID               string         `json:"product_id"`
+	Type                    string         `json:"type"`
+	ChannelCode             string         `json:"channel_code,omitempty"`
+	ReferenceID             string         `json:"reference_id"`
+	AccountIdentifier       string         `json:"account_identifier,omitempty"`
+	Currency                string         `json:"currency,omitempty"`
+	Amount                  float64        `json:"amount"`
+	NetAmount               float64        `json:"net_amount"`
+	Cashflow                string         `json:"cashflow"`
+	Status                  string         `json:"status"`
+	ChannelCategory         string         `json:"channel_category"`
+	BusinessID              string         `json:"business_id"`
+	SettlementStatus        string         `json:"settlement_status,omitempty"`
+	EstimatedSettlementTime *time.Time     `json:"estimated_settlement_time,omitempty"`
```

Comment:
📝 i confirmed on the test env that these two info exists on paid invoice transaction 
```json
{
    "has_more": false,
    "data": [
        {
            "id": "txn_10825083-d3e2-42b4-86e2-52930f730a88",
            "product_id": "6470821a1ace4e001adb5011",
            "type": "PAYMENT",
            "status": "SUCCESS",
            "channel_category": "CARDS",
            "channel_code": "VISA",
            "reference_id": "4b840e0d-ab1f-42a3-b8d9-bf19acee56db",
            "account_identifier": "400000XXXXXX1091",
            "currency": "IDR",
            "amount": 5000,
            "net_amount": 2620,
            "cashflow": "MONEY_IN",
            "settlement_status": "PENDING",
            "estimated_settlement_time": "2023-05-30T17:00:00.000Z",
            "business_id": "xxx",
            "created": "2023-05-26T09:55:39.890Z",
            "updated": "2023-05-26T09:55:39.890Z",
            "fee": {
                "xendit_fee": 2145,
                "value_added_tax": 235,
                "xendit_withholding_tax": 0,
                "third_party_withholding_tax": 0,
                "status": "PENDING"
            }
        }
    ],
    "links": []
}
```